sklearn/calibration.py

```diff
@@ -890,6 +904,8 @@ def grad(AB):
 
     AB0 = np.array([0.0, log((prior0 + 1.0) / (prior1 + 1.0))])
     AB_ = fmin_bfgs(objective, AB0, fprime=grad, disp=False)
+    # The tuned parameters are converted back to the original scale and offset.
+    AB_ /= scale_constant
```

Comment:
@ogrisel @lorentzenchr I think we might need to do this if we want to keep the values of the parameters same.